app/routes.js

```diff
@@ -45,6 +46,37 @@ export const routes = (authProvider, sessionManager) => {
     stream.on('close', () => res.end());
   }));
 
+  const regExp = new RegExp('^/organisation/[A-Za-z]\\d{6}-\\d{2}');
+
+  // eslint-disable-next-line consistent-return
```

Comment:
Rather than disable the ESLint warning we can just do `return next();`

Reply:
done